ios/StatusIm.xcodeproj/project.pbxproj

```diff
@@ -276,6 +269,20 @@
 			remoteGlobalIDString = EB2648DF1C7BE17A00B8F155;
 			remoteInfo = ReactNativeConfig;
 		};
+		AB46514D21E5C18D002B6CA6 /* PBXContainerItemProxy */ = {
+			isa = PBXContainerItemProxy;
+			containerPortal = AB46511921E5C18D002B6CA6 /* RNLanguages.xcodeproj */;
+			proxyType = 2;
+			remoteGlobalIDString = 134814201AA4EA6300B7C361;
+			remoteInfo = RNLanguages;
+		};
+		AB46514F21E5C18D002B6CA6 /* PBXContainerItemProxy */ = {
+			isa = PBXContainerItemProxy;
+			containerPortal = AB46511921E5C18D002B6CA6 /* RNLanguages.xcodeproj */;
+			proxyType = 2;
+			remoteGlobalIDString = 2D2A28471D9B043800D4039D;
+			remoteInfo = "RNLanguages-tvOS";
```

Comment:
Probably we don't need to add tvOS related stuff

Reply:
This gets injected automatically by xcode and it seems like we have tvOS config in this file everywhere so I kept it consistent. Happy to remove it if we prefer that way though!